delfin/drivers/dell_emc/vnx/vnx_block/consts.py

```diff
@@ -24,8 +24,8 @@
 EXCEPTION_MAP = {CER_ERR: exception.SSLCertificateFailed,
                  CALLER_ERR: exception.InvalidUsernameOrPassword,
                  SECURITY_ERR: exception.InvalidUsernameOrPassword,
-                 TRYING_CONNECT_ERR: exception.ConnectTimeout,
-                 CONNECTION_ERR: exception.ConnectTimeout,
+                 TRYING_CONNECT_ERR: exception.InvalidIpOrPort,
```

Comment:
What is there is some network connectivity issue? It may not be invalid IP or PORT

Reply:
Page display requirements

Reply:
Ok hope the message is displayed accordingly